Packs/XMCO/Classifiers/classifier-mapper-incoming-XMCO.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id": "XMCO - Serenety Mapper",
+    "name": "XMCO - Serenety Mapper",
+    "type": "mapping-incoming",
+    "description": "Maps incoming XMCO alert fields.",
+    "fromVersion": "6.10.0",
+    "mapping": {
+        "XMCO Serenety Alert": {
+            "dontMapEventToLabels": false,
+            "internalMapping": {
+                "Serenety category": {
+                    "complex": null,
+                    "simple": "category"
+                },
+                "Serenety subcategory": {
```

Comment:
Please use the OOTB field `Sub Category`. No need to create dedicated fields that are already existing. Hence, you can remove the added incident field `Serenety subcategory` from the PR.

Reply:
Our developpemen XSOAR version is 6.12, not 8. The sub category field is nested in the field CustomFields.

Is this the same on the XSOAR v8 ?

Reply:
This field that I've mentioned `Sub Category` is located in the commonTypes which is installed on both xsoar v6 and v8 versions.